docs/install.md

```diff
@@ -67,6 +70,44 @@ and then install the optimisation variant:
 
     pip install gadopt[optimisation]
 
+## Using the G-ADOPT library on Gadi
+
+The G-ADOPT team maintains an up-to-date G-ADOPT and Firedrake installation
+with all optional dependencies on NCI's Gadi HPC system. To use G-ADOPT on 
+Gadi, create an NCI account, then follow 
+[this link](https://my.nci.org.au/mancini/project/fp50/join) to request to 
+join the fp50 project. Once your request is approved, you can
+access G-ADOPT on Gadi by running:
+
+    module use /g/data/fp50/modules
+    module load g-adopt
+
+To use G-ADOPT in your batch scripts, you'll need to add the following to your
+PBS flags:
+
+    -lstorage=gdata/fp50
+
+By default, this will give you the latest release of G-ADOPT, and load the
+most recent `firedrake` module. If you wish to use your own G-ADOPT installation, set
+the environment variable `MY_GADOPT` and load the firedrake module.
+
+    export MY_GADOPT=/g/data/ab12/path/to/my/gadopt
+    module use /g/data/fp50/modules
+    module load firedrake
+
+You can also use G-ADOPT in a JupyterLab session on the
+[ARE](https://opus.nci.org.au/spaces/Help/pages/162431120/ARE+User+Guide) at 
+NCI. When launching a JupyterLab session, add `gdata/fp50` to the storage 
+field and the following in the 'Advanced options' section:
+
+![G-ADOPT ARE](images/g-adopt-are.png){: style="width:450px"}
```

Comment:
Can you change the screenshot to show that you need to load the new g-adopt module (and probably dont need to pass MY_GADOPT as default) 

![image](https://github.com/user-attachments/assets/d79614a4-f810-4449-958b-9cd79a85bcd2)